apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/style.scss

```diff
@@ -0,0 +1,134 @@
+.main-navigation {
```

Comment:
We may want to add the expected block classname `wp-block-a8c-navigation-menu` in the rendered output to make these a bit more specific.

(Note that if a non-dynamic block is added this classname is serialized for you. The server render for dynamic blocks return exactly what we specify so this classname is missing here.)